EQTool/UI/BaseSaveStateWindow.cs

```diff
@@ -192,15 +195,35 @@ protected void MaximizeWindow(object sender, RoutedEventArgs e)
         {
             WindowState = WindowState == System.Windows.WindowState.Maximized ? System.Windows.WindowState.Normal : System.Windows.WindowState.Maximized;
         }
-
+        
         protected void HoverZone_MouseEnter(object sender, MouseEventArgs e)
         {
-            baseViewModel.IsMouseOverTitleArea = true;
+            hoverDebounceTs?.Cancel();
+            hoverDebounceTs = new CancellationTokenSource();
+            var debounceToken = hoverDebounceTs.Token;
+
+            Task.Run(async () =>
```

Comment:
You cant use task.run like this unless the executing code for the IsMouseOverTitleArea runs in the UI thread,
Since tasks have no guarantee if they run in the calling thread or a different this this will cause random runtime errors when the task is ran in a thread other than the UI thread.

You can either make this function async void and remove the task.run (I think this will work)
Or, you can add the IAppDispatcher to the ctor of basesavestatewindow and use that to wrap the call to the setting like
<img width="616" height="328" alt="image" src="https://github.com/user-attachments/assets/289e90e1-e338-486d-ba18-a9141640d1bb" />